WcaOnRails/app/models/competition.rb

```diff
@@ -48,6 +51,7 @@ class Competition < ActiveRecord::Base
     cellName
     showAtAll
     isConfirmed
+    eventSpecs
```

Comment:
I've created issue #886 to make sure we don't forget to remove the `eventSpecs` column once this PR has been implemented.